### ISSUE TITLE
Clean F*'s build stdout and stderr

### ISSUE THREAD

**Issue by cpitclaudel** [FStarLang/FStar#1273]
That would make it easier to see the important stuff.  It should be easy for each of us to check what output comes from their parts of the code.

**Comment by cpitclaudel** [comment]
Added @victor-dumitrescu to have a look at the warnings from _cache/ in native-tactics

**Comment by mtzguido** [comment]
The warnings about Canon.ml are due to (seemingly incomplete) pattern matches in the extracted projectors. This happens for any other match where totality is non-trivial.

Solutions are to
1) kill that warning: we're doing totality checks ourselves
2) add a catch all case at the end, but then we need to check whether or not it is "OCaml-redundant", or we get a new warning.

I'm vouching for (1), and there's other warnings that we wanted to kill too.

**Comment by cpitclaudel** [comment]
Yup, option 1 is good. That's what I already did in _tags for other extracted files – it just needs to be extended.

**Comment by mtzguido** [comment]
Ah, I forgot about that, but I think it would be better to include the ignore within the extracted file (it was some syntax like `[@@ warning -8]`) so everyone benefits from it.
